src/org/omegat/ColorScheme_dark.properties

```diff
@@ -0,0 +1,40 @@
+# OmegaT application default colors scheme
+# This is a table of default colors definitions when no customization.
+# Here is a definition for dark theme, such as GTK LAF(dark), OSX Mojava and 3rd party dark LAFs.
```

Comment:
I think people understand the concept of a dark color scheme, so I would simplify the description and instead document the format of the expected values:

```suggestion
# OmegaT default dark color scheme
#
# Format is 8-bit RGB in hexadecimal (like standard HTML colors without the
# leading '#')
```